testdata/baselines/reference/submodule/compiler/intersectionPropertyCheck.js

```diff
@@ -22,11 +22,22 @@ function test<T extends object>(value: T): Test {
 
 
 //// [intersectionPropertyCheck.js]
+var __assign = (this && this.__assign) || function () {

```

Comment:
Aside: If our min target is truly supposed to be `es2015`, this helper should never be emitted (`Object.assign` is used directly instead), but our tests don't default to that yet, so I err'd on keeping the helper output to minimize diffs.